README.md

```diff
@@ -66,14 +68,22 @@ If you want to experience a hands-free ZITADEL, you should use [ZITADEL Cloud](h
 It is free for up to 25'000 authenticated requests and provides you all the features that make ZITADEL great.
 Learn more about the [pay-as-you-go pricing](https://zitadel.com/pricing).
 
+### Example applications
+
+Clone one of our [example applications](https://zitadel.com/docs/examples/introduction#clone-a-sample-project) or deploy them directly to Vercel.
+
+### SDKs
+
+Use our [SDKs](https://zitadel.com/docs/examples/sdks) for your favorite language and framework.
+
 ## Why choose ZITADEL
 
-We built ZITADEL with a complex multi-tenancy architecture in mind and provide the best solution to handle B2B customers and partners.
-Yet it offers everything you need for a customer identity (CIAM) use case.
+We built ZITADEL with a complex multi-tenancy architecture in mind and provide the best solution to handle [B2B customers and partners](https://zitadel.com/docs/guides/solution-scenarios/b2b).
+Yet it offers everything you need for a customer identity ([CIAM](https://zitadel.com/docs/guides/solution-scenarios/b2c)) use case.
```

Comment:
```suggestion
Yet it offers everything you need for a customer identity [CIAM](https://zitadel.com/docs/guides/solution-scenarios/b2c) use case.
```

Reply:
... customer identity (CIAM) use case ... 

I think for that is was fine. We can also just use CIAM.

Reply:
... customer identity (CIAM) use case ... 

I think for that is was fine. We can also just use CIAM.